custom_components/mikrotik_router/coordinator.py

```diff
@@ -1553,14 +1583,23 @@ def get_firmware_update(self) -> None:
 
         if self.ds["fw-update"]["installed-version"] != "unknown":
             try:
-                self.major_fw_version = int(
```

Comment:
this block change is most likely not necessery, as it is used to control only capsman selection, which is erroneous.

Reply:
I changed this to add self.minor_fw_version so that get_capsman_hosts can check the minor version > 12. In verison 7.13 the capsman path is no longer /caps-man... and the registration table in under /interface/wifi/registration-table.